src/index.js

```diff
@@ -1,45 +1,42 @@
 import './style.css';
+import { Task, getData, saveData } from './task.js';
+import UI from './ui.js';
+import display from './display.js';
 
-const dataToDoList = [
-  {
-    description: 'Jumping',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Running',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Eating',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Drinking',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Sleeping',
-    completed: true,
-    index: 4,
-  },
-];
-function populateList(tasks) {
-  const containerList = document.querySelector('.todo');
-  let listHTML = '';
-  tasks.forEach((task) => {
-    listHTML += `
-      <li class="list-to-do">
-        <input type="checkbox" class="to-do">
-        ${task.description}
-        <i class="fa-solid fa-trash"></i>
-      </li>
-      <hr class="line">
-    `;
-  });
-  containerList.innerHTML = listHTML;
+let tasksList;
+if (getData() === null) {
+  tasksList = [];
+} else {
+  tasksList = getData();
 }
-populateList(dataToDoList);
+
+const addTask = (newTask) => {
+  let index;
+  if (getData() === null) {
+    index = 1;
+  } else {
+    tasksList = getData();
+    index = tasksList.length + 1;
+  }
+  const task = new Task(newTask, false, index);
```

Comment:
- [ ] there's a small bug where the index is being saved to the completed key in the object, kindly fix it 🙏 

![image](https://user-images.githubusercontent.com/89282221/216789728-336a9eeb-824e-4583-a8c0-8c19a59c2f51.png)